acceptance-tests/features/create_new_case.feature

```diff
@@ -4,12 +4,12 @@ Feature: Create case and deed helper page
   to have a page that will let them easily create a deed and a case for test purposes
 
 Scenario: Go to the create case page
-    Given I navigate to the case list page
-    When I click on the 'Create a new case' button
-    Then I am taken to the 'Create a new case' page
+    Given I navigate to the conveyancer frontend "/cases" page
+    When I click on the "Create a new case" link
```

Comment:
@michaeldfallen not 100% sure but since this file I assume goes through the ruby interpreter we might be better off with single quotes as double quotes are slower as they allow string interpolation. Again not sure, thoughts?